addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6887,7 +6973,7 @@ registry.ImageTools = ImageHandlerOption.extend({
         }
 
         // Add shape animations on hover.
-        if (img.dataset.hoverEffect && !this._isDeviceShape() && !this._isAnimatedShape()) {
+        if (img.dataset.hoverEffect && !this._isDeviceShape() && this.shapeAnimationSpeed === undefined) {
```

Comment:
hover effect animations  no longer work, I think it is because of this change...